### PR TITLE
Courses

### PR DESCRIPTION
**Pull request by jonahbroyer**
check out my seqelize!

### CODE REVIEW

**Review by SamsonDeVol — CHANGES_REQUESTED** [review]
remove .env

File: lib/sequelize.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { Sequelize } = require('sequelize')
+const sequelize = new Sequelize({
+    dialect: 'mysql',
+    host: process.env.MYSQL_HOST || 'localhost',
+    port: process.env.MYSQL_PORT || 3306,
+    database: process.env.MYSQL_DB_NAME,
```
Comment (swearingenmj) [inline lib/sequelize.js:6]: make sure the variables match the .env file

File: models/course.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const sequelize = require('../lib/sequelize')
+const { User } = require('./user')
+
+const Course = sequelize.define('course', {
+  id: { type: DataTypes.INTEGER, allowNull: false },
+  subject_code: { type: DataTypes.STRING, allowNull: false },
+  number: { type: DataTypes.STRING, allowNull: false },
+  title: { type: DataTypes.STRING, allowNull: false },
+  term: { type: DataTypes.STRING, allowNull: false },
+  instructor_id: { type: DataTypes.INTEGER, allowNull: false }
```
Comment (swearingenmj) [inline models/course.js:12]: lets go over this together. we should set some to NOT NULL and include foreign keys

File: models/course.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+})
+
+/*
+* Set up one-to-many relationship between Course and User.
+*/
+Course.hasMany(User, { foreignKey: { allowNull: false } })
+User.belongsTo(Course)
```
Comment (swearingenmj) [inline models/course.js:19]: add foreign key

**Review by swearingenmj — DISMISSED** [review]
see comments

**Review by SamsonDeVol — APPROVED** [review]

**Pull request merged**